consensus/votecounter/ballot.go

```diff
@@ -0,0 +1,41 @@
+package votecounter
+
+import "github.com/NethermindEth/juno/consensus/types"
+
+type VoteType uint8
+
+const (
+	Prevote VoteType = iota
+	Precommit
+)
+
+type ballot []bool
+
+type ballotSet[A types.Addr] struct {
+	ballots     map[A]ballot
+	perVoteType []types.VotingPower
```

Comment:
The same applies here for `perVoteType`. By giving static sizes we give the compiler and the runtime enough knowledge to perform optimizations in both stages. In this case, I believe quite relevant too